lib/routes/account.js

```diff
@@ -873,7 +874,7 @@ module.exports = function (
             id: isA.string().length(32).regex(HEX_STRING).required(),
             isCurrentDevice: isA.boolean().required(),
             lastAccessTime: isA.number().min(0).required().allow(null),
-            name: isA.string().max(255).required(),
+            name: isA.string().max(255).regex(DISPLAY_SAFE_UNICODE).required(),
```

Comment:
Nice catch @philbooth, we probably don't need to validate it in the response schema here (or if we do, we need to do something to ensure the devices list can still be read)